src/lazy/binary/raw/v1_1/immutable_buffer.rs

```diff
@@ -1317,8 +1317,9 @@ mod tests {
         encode_macro_fn: impl FnOnce(MacroAddress) -> Vec<u8>,
         test_fn: impl FnOnce(BinaryEExpArgsIterator_1_1<'_>) -> IonResult<()>,
     ) -> IonResult<()> {
-        let mut context = EncodingContext::empty();
-        let template_macro = TemplateCompiler::compile_from_text(context.get_ref(), macro_source)?;
+        let mut context = EncodingContext::for_ion_version(IonVersion::v1_1);
```

Comment:
🪧 At first, Ion 1.0 and 1.1 readers would use the same `EncodingContext`. Later, `EncodingContext` was modified to be able to take an Ion version, allowing it to initialize the symbol table differently to accommodate Ion 1.1's new symtab. This PR continues that differentiation, causing 1.0 encoding contexts to be initialized with an empty macro table.